quetz/cli.py

```diff
@@ -591,8 +594,11 @@ def watch_job_queue(
 
     config_file = _get_config(path)
 
+    if not os.environ.get('QUETZ_CONFIG_FILE'):
+        os.environ['QUETZ_CONFIG_FILE'] = config_file
+
     config = Config(config_file)
-    manager = SubprocessWorker("", {}, config)
+    manager = SubprocessWorker("", {}, config, {'max_workers': num_procs})
```

Comment:
ultimately, this will be configured in [workers] section of config.toml. But for the moment only SubprocessWorker is supported ;-)

Reply:
sure!